src/shipit_code_coverage/shipit_code_coverage/codecov.py

```diff
@@ -150,14 +155,15 @@ def go(secrets, client_id=None, client_token=None):
     task_data = taskcluster.get_task_details(task_id)
     revision = task_data['payload']['env']['GECKO_HEAD_REV']
     logger.info('Revision %s' % revision)
-    commit_sha = get_github_commit(revision)
-    logger.info('GitHub revision %s' % commit_sha)
 
     download_coverage_artifacts(task_id)
 
     clone_mozilla_central(revision)
     build_files()
 
+    commit_sha = get_github_commit(revision)
+    logger.info('GitHub revision %s' % commit_sha)
```

Comment:
Nit: the cli_common logger (that you're using) support [structlog](https://structlog.readthedocs.io/en/stable/).
So you can do this:
```python
logger.info('GitHub revision', commit=commit_sha)
```